db/migrate/20160502231050_create_products.rb

```diff
@@ -0,0 +1,14 @@
+class CreateProducts < ActiveRecord::Migration
+  def change
+    create_table :products do |t|
+      t.string :name, null: false
+      t.integer :price, null: false
+      t.integer :user_id, null: false
+      t.integer :stock, null: false
+      t.string :category
+      t.string :description
+      t.string :status, null: false
+      t.timestamps null: false
```

Comment:
I think `null: false` is doing the same thing that you're doing in your models where you have something like, `validates :name, presence: true` So I would do one or the other.